cmd/sealer/cmd/cluster/join.go

```diff
@@ -19,6 +19,9 @@ import (
 	"io/ioutil"
 	"path/filepath"
 
+	imagecommon "github.com/sealerio/sealer/pkg/define/options"
```

Comment:
usually, there should be three groups separated by blank for go-imports
1. the golang official package
2. the package for the current project
3. the others